webviz_subsurface/plugins/_tornado_plotter_fmu/tornado_plotter_fmu.py

```diff
@@ -0,0 +1,282 @@
+from typing import Dict, List
+from pathlib import Path
+import json
+
+import dash
+from dash.dependencies import Input, Output, ALL
+import dash_html_components as html
+import dash_core_components as dcc
+from webviz_config import WebvizPluginABC
+from webviz_config import WebvizSettings
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+import webviz_core_components as wcc
+
+import webviz_subsurface
+from webviz_subsurface._providers import EnsembleTableProviderFactory
+from webviz_subsurface._components.tornado.tornado_widget import TornadoWidget
+from webviz_subsurface._datainput.fmu_input import find_sens_type
+
+
+class TornadoPlotterFMU(WebvizPluginABC):
+    """General tornado plotter for FMU data from a csv file of responses.
+    ---
+    * **`ensembles`:** Which ensembles in `shared_settings` to visualize.
+    * **`csvfile`:** Relative ensemble path to csv file with responses
+    * **`aggregated_csvfile`:** Alternative to ensemble + csvfile with
+    aggregated responses. Requires REAL and ENSEMBLE columns
+    * **`aggregated_parameterfile`:** Necessary when aggregated_csvfile
+    is specified. File with sensitivity specification for each realization.
+    Requires columns REAL, ENSEMBLE, SENSNAME and SENSCASE.
+    * **`initial_response`:** Initialize plugin with this response column
+    visualized
+    * **`single_value_selectors`:** List of columns in response csv file
+    that should be used to select/filter data. E.g. for UNSMRY data the DATE
+    column can be used. For each entry a Dropdown is shown with all unique
+    values and a single value can be selected at a time.
+    * **`multi_value_selectors`:** List of columns in response csv file
+    to filter/select data. For each entry a Select is shown with
+    all unique values. Multiple values can be selected at a time,
+    and a tornado plot will be shown from the matching response rows.
+    Used e.g. for volumetrics data, to select a subset of ZONES and
+    REGIONS.
+    """
+
+    # pylint: disable=too-many-locals, too-many-arguments
+    def __init__(
+        self,
+        app: dash.Dash,
+        webviz_settings: WebvizSettings,
+        csvfile: str = None,
+        ensemble: str = None,
+        aggregated_csvfile: Path = None,
+        aggregated_parameterfile: Path = None,
+        initial_response: str = None,
+        single_value_selectors: List[str] = None,
+        multi_value_selectors: List[str] = None,
+    ):
+        super().__init__()
+        WEBVIZ_ASSETS.add(
+            Path(webviz_subsurface.__file__).parent
+            / "_assets"
+            / "css"
+            / "container.css"
+        )
```

Comment:
This can be removed after standardizing the layout?

Reply:
yes, I will update it when the layout PR is merged